iaso/tests/api/org_unit_change_requests/test_org_unit_change_requests.py

```diff
@@ -737,10 +737,14 @@ def get_reference_instance_ids(instances):
         expected_row_data = ["" if v is None else str(v) for v in expected_row_data]
         first_data_row = ["" if v is None else str(v) for v in first_data_row]
 
-        self.assertEqual(
-            first_data_row,
-            expected_row_data,
-        )
+        # Sort instance IDs in both lists before comparison
+        for i, (expected, actual) in enumerate(zip(expected_row_data, first_data_row)):
+            if "," in expected and "," in actual:  # Check if this is a comma-separated list of IDs
+                expected_sorted = ",".join(sorted(expected.split(",")))
+                actual_sorted = ",".join(sorted(actual.split(",")))
+                self.assertEqual(expected_sorted, actual_sorted, f"Difference at index {i}")
+            else:
+                self.assertEqual(expected, actual, f"Difference at index {i}")
```

Comment:
I already saw this in another PR and I also fixed it in a different way (see https://github.com/BLSQ/iaso/commit/d6dad372af1fc3b989e3a2189177ea9508909622 and https://github.com/BLSQ/iaso/commit/2712676815720039aae35eb640eed13425bbe21b)

Is this an error?

Reply:
i will remove it thanks